extensions/amp-base-carousel/0.1/amp-base-carousel.js

```diff
@@ -159,6 +159,11 @@ class AmpCarousel extends AMP.BaseElement {
     return isLayoutSizeDefined(layout);
   }
 
+  /** @override */
+  prerenderAllowed() {
```

Comment:
Should this be enabled in the extensions that extend base carousel instead of here? It makes it more difficult to analyze if they inherit this privilege.

Reply:
I'm slightly puzzled how that works. How do I find these extensions?

Reply:
Oh, I thought they extended this directly. But, it appears they just compose it. So this should  be fine.

Reply:
Approved.